drivers/gles3/shaders/copy.glsl

```diff
@@ -68,12 +68,27 @@ uniform highp mat4 pano_transform;
 uniform highp vec4 asym_proj;
 #endif
 
+// These definitions are here because the shader-wrapper builder does
+// not understand `#elif defined()`
+#ifdef USE_TEXTURE3D
```

Comment:
That's weird, how does checking this fix the parsing of `#elif defined()`?

Reply:
The Python file generating C++ headers from .glsl files does not understand this.

When it sees `#ifdef XXX` it will add `XXX` to an enum in the wrapping class.

When only the `#elif defined(USE_TEXTURE...` is used, there is no enum variant generated for this, so it's impossible to set from the rasterizer implementation.

These lines solve this by making the conditionals known to the generator, so they appear in the C++ class, so they can be used inside the renderer :D